app/assets/javascripts/test/sagas/skeletontracing_saga.spec.js

```diff
@@ -342,20 +343,25 @@ test("SkeletonTracingSaga should emit update actions on split tree", t => {
   const updateActions = testDiffing(testState.tracing, newState.tracing, newState.flycam);
   t.is(updateActions[0].name, "createTree");
   t.is(updateActions[0].value.id, 3);
+
   t.is(updateActions[1].name, "createNode");
-  t.is(updateActions[1].value.id, 4);
+  t.is(updateActions[1].value.id, 2);
```

Comment:
Not sure about all these changes, but as you said, I'll test in the tracing view once you give the go :)